maltoolbox/language/languagegraph.py

```diff
@@ -19,9 +19,9 @@
 @dataclass
 class LanguageGraphAsset:
     name: str = None
-    associations: list[LanguageGraphAssociation] = None
-    attack_steps: list[LanguageGraphAttackStep] = None
-    description: dict = None
+    associations: list[LanguageGraphAssociation] = field(default=None, repr=False)
+    attack_steps: list[LanguageGraphAttackStep] = field(default=None, repr=False)
+    description: dict = field(default=None, repr=False)
     # MAL languages currently do not support multiple inheritance, but this is
     # futureproofing at its most hopeful.
     super_assets: list = None
```

Comment:
The fields that are initialized to None should either be marked with Optional or initialized to empty dicts/lists with field()